app/config.py

```diff
@@ -93,6 +114,24 @@ class QueueNames(object):
     # Queue for delivery receipts such as emails sent through AWS SES.
     DELIVERY_RECEIPTS = "delivery-receipts"
 
+    DELIVERY_QUEUES = {
+        "sms": {
+            Priorities.LOW: SEND_SMS_LOW,
+            Priorities.MEDIUM: SEND_SMS_MEDIUM,
+            Priorities.HIGH: SEND_SMS_HIGH,
+        },
+        "email": {
+            Priorities.LOW: BULK,
+            Priorities.MEDIUM: SEND_EMAIL,
+            Priorities.HIGH: PRIORITY,
+        },
+        "letter": {
```

Comment:
can we remove this ?

Reply:
I wonder if this is meant for tests maybe? Else I'd vote to not have it too.

Reply:
😞 yeah, exactly, a ton of tests fail without this.